scapy/layers/bluetooth.py

```diff
@@ -736,7 +833,7 @@ class HCI_Cmd_LE_Long_Term_Key_Request_Reply(Packet):
 class HCI_Event_Hdr(Packet):
     name = "HCI Event header"
     fields_desc = [XByteField("code", 0),
-                   ByteField("length", 0), ]
+                   LenField("len", None, fmt="B"), ]
```

Comment:
(Just to be sure: this field only counts payload. Is that expected ?)

Reply:
Yup, that's fine! Poking at some packets captured by my machine, it looks like they are also formatted this way.

Reply:
Great ! Usually people think it also includes the current layer's length, I was just checking